docs/examples.md

```diff
@@ -78,4 +78,106 @@
 
 2. Curl it via localhost
 
-    `curl localhost:8082/`
\ No newline at end of file
+    `curl localhost:8082/`
+
+## Local insecure registry
+
+This guide takes you through setting up a local insecure (http) registry and integrating it into your workflow so that:
+- you can push to the registry from your host
+- the cluster managed by k3d cann pull from that registry
+
+The registry will be named `registry.local` and run on port `5000`.
+### Create the registry
+
+<pre>
+docker volume create local_registry
+
+docker container run -d --name <b>registry.local</b> -v local_registry:/var/lib/registry --restart always -p <b>5000:5000</b> registry:2
+</pre>
+
+### Create the cluster with k3d
+
+First we need a place to store the config template: `mkdir -p /home/${USER}/.k3d`
+
+Create a file named `config.toml.tmpl` in `/home/${USER}/.k3d`, with following content:
+
+<pre>
```

Comment:
Maybe you can leave comments in this config block, telling the reader, that the first part is the original part, which was already there and that only the lower part has to be added?

Reply:
Added comment to the <pre>-block